### PR TITLE
Fix inline doc typo

### PR DESCRIPTION
**Pull request by LeoColomb**
Just a silly typo 😄

### CODE REVIEW

**Comment by LeoColomb** [comment]
I can bump the chart, but should be ok to bypass, I guess. Let me know.

**Comment by RangerRick** [comment]
> I can bump the chart, but should be ok to bypass, I guess. Let me know.

might as well wait until the script persistence is merged so we can fix all 3

**Review by RangerRick — APPROVED** [review]

**Pull request merged**
